src/promptflow/tests/executor/e2etests/test_executor_validation.py

```diff
@@ -204,6 +204,19 @@ def test_flow_run_input_type_invalid(self, flow_folder, line_input, error_class,
         with pytest.raises(error_class):
             executor.exec_line(line_input)
 
+    def test_flow_run_input_assignment(self, dev_connections):
```

Comment:
We can separate this test case into two, with well naming to indicate their intents.
i.e.
* test_invalid_flow_run_inputs_should_not_saved_to_run_info()
* test_valid_flow_run_inpust_should_saved_to_run_info()